terraform/bigquery.tf

```diff
@@ -312,7 +312,31 @@ resource "google_bigquery_table_iam_member" "failure_events_viewers" {
   member     = each.value
 }
 
-// Unique Events View 
+// Base Views
+
+resource "google_bigquery_table" "base_views" {
+  for_each = fileset("${path.module}/data/bq_views/base", "*")
```

Comment:
Should this be `*.sql`?

Reply:
yes, fixed.

Reply:
I dont think you committed this change or pushed it up.

Reply:
hmm strange I tried amending the branch and force pushing, but it didn't appear to take. I just made a separate commit instead.